package.json

```diff
@@ -101,10 +101,13 @@
     "serve": "11.0.0",
     "shelljs": "0.7.8",
     "shx": "0.2.2",
-    "sinon": "~4.5.0"
+    "sinon": "~4.5.0",
+    "tslint": "5.18.0"
```

Comment:
Is this dependency just for our own usage now?

Reply:
Yes